### PR TITLE
tss2 *: Enhance Fapi Tools integration tests

### PR DESCRIPTION
**Pull request by cplappert**
This PR enhances the tss2 tools integration tests. Enhancements are:
1. The tests now additionally run with the ECC default profile.
2. The tests now test input boundaries with empty and big input files. Because of this, the TSS version in the CI needs to be updated to include latest fixes so that tests do not fail.

### CODE REVIEW

File: .ci/download-deps.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 function get_deps() {
 
-	TSS_VERSION=${TPM2_TSS_VERSION:-2.4.0}
+	TSS_VERSION=${TPM2_TSS_VERSION:-2.4.x}
```
Comment (tstruk) [inline .ci/download-deps.sh:6]: 2.4.x is a branch name. Don't you want a tag here? You may use 2.4.1 instead.

Comment (cplappert) [reply .ci/download-deps.sh:6]: The CI errors with `tss2 pcrextend` will be fixed as soon as https://github.com/tpm2-software/tpm2-tss/pull/1811 and https://github.com/tpm2-software/tpm2-tss/pull/1812 are merged. Thus, for the tests to run successfully, a version of the TSS is needed where these fixes are included. Probably 2.4.2?

Comment (tstruk) [reply .ci/download-deps.sh:6]: Yes, it will be in 2.4.2, which should be out soon.

Comment (cplappert) [reply .ci/download-deps.sh:6]: Ok. I will update the commit as soon as it is available.

Comment (cplappert) [reply .ci/download-deps.sh]: So, I updated to tag 2.4.2. Travis is fine now, but Cirrus reports a weird error. Maybe it was only temporary. Could retrigger the build?

Comment (tstruk) [reply .ci/download-deps.sh]: On Cirrus it always runs against TSS master so it doesn't really matter what TSS version is set in .ci/download-deps.sh
but it should work just fine, as it also runs fine on Travis against TSS master: https://travis-ci.org/github/tpm2-software/tpm2-tools/jobs/715117614
We have recently checked in a change to tcti-mssim to change it to async mode. Maybe this causes some trouble in Cirrus. I will look into it.

Comment (cplappert) [reply .ci/download-deps.sh]: Anything new?

**Comment by williamcroberts** [comment]
Since cirrus is in general failing, I will merge this while @tstruk sorts out cirrus

**Pull request merged**